src/vs/workbench/contrib/positronConnections/browser/components/schemaNavigationActionBar.tsx

```diff
@@ -73,38 +55,24 @@ export const ActionBar = (props: React.PropsWithChildren<ConnectionActionBarProp
 					<ActionBarRegion location='left'>
 						<ActionBarButton
 							align='left'
-							// TODO: should have a connect-icon
-							iconId='positron-new-connection'
-							tooltip={() => 'Connect'}
-							disabled={connectDisabled}
-							onPressed={() => props.selectedEntry?.connect?.()}
+							iconId='arrow-left'
+							tooltip={() => localize('positron.schemaNavigationActionBar.back', 'Back')}
```

Comment:
Should be:

```
tooltip={(() => localize('positron.schemaNavigationActionBar.back', 'Back'))()}
```

Reply:
Resolved in d28f65b5881b7f2035f17782db70a57dd6cac3f3